### DISCUSSION TITLE
I cannot get outputs 3-8 to work in Reaper with the main VST3 variant.

### DISCUSSION THREAD

**Discussion by GBaige** [DISTRHO/Cardinal#430] [Q&A] [answered]
I'm trying to route audio from Cardinal to 8 different tracks in Reaper to process with different effects, using the Audio 8 module. Only the first two outputs work.

E.g. Cardinal is on track 1. Track 1 sends mono audio from input 1 on the Audio 8 plugin to track 2, mono from input 2 to track 3, mono from input 3 to track 4, and so on. I have a VCO's output patched to input 3. Try as I might, I cannot get the audio through to track 4.

What am I doing wrong?

**Comment by falkTX** [comment, marked as answer]
vst3 works in an odd way for this. you might have more luck with clap and using latest experimental reaper builds

**Reply by dromer** [reply]
Or LV2 and any recent Reaper release.
